src/Symfony/Component/HttpClient/Psr18Client.php

```diff
@@ -195,6 +198,19 @@ public function reset(): void
             $this->client->reset();
         }
     }
+
+    protected function getReasonPhrase(HttpClientResponseInterface $response): ?string
```

Comment:
```suggestion
    private function getReasonPhrase(HttpClientResponseInterface $response): ?string
```

This is not an extension point.